metadata-ingestion/src/datahub/ingestion/source/sql_queries.py

```diff
@@ -67,28 +73,19 @@ class SqlQueriesSourceConfig(PlatformInstanceConfigMixin, EnvConfigMixin):
         default=None,
     )
     override_dialect: Optional[str] = Field(
-        description="The SQL dialect to use when parsing queries. Overrides automatic dialect detection.",
+        description="DEPRECATED: This field is ignored. SQL dialect detection is now handled automatically by the SQL parsing aggregator based on the platform.",
```

Comment:
it might make sense to preserve this functionality (e.g. add it to the aggregator). iirc we've had a couple cases where our automatic detection doesn't produce the right sqlglot dialect name, and having the override was important

Reply:
addressed in https://github.com/datahub-project/datahub/pull/14201